src/pages/Contents/Sidebar/Sidebar.js

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import { Link } from "react-router-dom";
+import { SidebarItems } from "./SidebarItems";
+
+export const Sidebar = () => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const toggleSidebar = () => {
+    setIsOpen(!isOpen);
+  };
+
+  return (
+    <SidebarContainer>
+      <SidebarMenuIcon>
+        <SidebarToggle
+          onClick={toggleSidebar}
+          alt="menu"
+          src="/images/Contents/Icons/bars-solid.png"
+        />
+        <CloseButton
+          onClick={toggleSidebar}
+          isOpen={isOpen}
+          alt="X"
+          src="/images/Contents/Icons/xmark-solid.png"
+        />
+      </SidebarMenuIcon>
+      <SidebarMenu isOpen={isOpen}>
+        <SidebarMenuIndex isOpen={isOpen}>목차</SidebarMenuIndex>
+        <LectureName isOpen={isOpen}>강좌명</LectureName>
+        <DivisionLine isOpen={isOpen} />
+        <SidebarItems isOpen={isOpen} toggleSidebar={toggleSidebar} />
+      </SidebarMenu>
+    </SidebarContainer>
+  );
+};
+
+const SidebarContainer = styled.div`
+  height: 100%;
+  max-width: 500px;
+  overflow: auto;
+  overflow-x: hidden;
+`;
+const SidebarMenuIcon = styled.div`
+  height: 50px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: #efefef;
+`;
+const SidebarToggle = styled.img`
+  width: 20px;
+  height: 20px;
+  margin-left: 15px;
+  &:hover {
+    cursor: pointer;
+  }
+`;
+const CloseButton = styled.img`
+  display: ${({ isOpen }) => (isOpen ? "" : "none")};
```

Comment:
```suggestion
  display: ${({ isOpen }) => (isOpen ? "" : "none")};
```
display 속성값으로 "" 이렇게 빈 스트링은 없습니다.
조건에 따라 block, inline, inline-block 같은 속성 화면에 display 속성이 보이거나, 안보이게 하는 값을 줘야 맞겠네요

Reply:
🪄 넵